<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>971</width>
    <height>806</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>150</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Методы грамматического разбора</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_8">
   <item>
    <widget class="QGroupBox" name="grboxMenu">
     <property name="title">
      <string/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="lblName">
          <property name="font">
           <font>
            <family>Century Gothic</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>&lt;AlgName&gt;</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="btnInform">
            <property name="minimumSize">
             <size>
              <width>300</width>
              <height>41</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Century Gothic</family>
              <pointsize>9</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Информация</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnParse">
            <property name="minimumSize">
             <size>
              <width>300</width>
              <height>41</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Century Gothic</family>
              <pointsize>9</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Наглядный разбор</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="grboxParse">
     <property name="title">
      <string/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QGroupBox" name="groupBox_2">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>111</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>Century Gothic</family>
                <pointsize>9</pointsize>
               </font>
              </property>
              <property name="title">
               <string>Введите строку для разбора:</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_7">
               <item>
                <widget class="QLineEdit" name="lineInpStr"/>
               </item>
               <item>
                <widget class="QPushButton" name="btnStart">
                 <property name="minimumSize">
                  <size>
                   <width>150</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>170</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="font">
                  <font>
                   <family>Century Gothic</family>
                   <pointsize>9</pointsize>
                  </font>
                 </property>
                 <property name="layoutDirection">
                  <enum>Qt::LeftToRight</enum>
                 </property>
                 <property name="autoFillBackground">
                  <bool>false</bool>
                 </property>
                 <property name="text">
                  <string>Разобрать</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="genBox">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>45</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>Century Gothic</family>
                <pointsize>9</pointsize>
               </font>
              </property>
              <property name="title">
               <string>Сгенерировать строку для разбора:</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_7">
               <item>
                <widget class="QPushButton" name="btnGenCor">
                 <property name="text">
                  <string>Корректная</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="btnGenIncor">
                 <property name="text">
                  <string>Некорректная</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="btnGenRand">
                 <property name="text">
                  <string>Произвольная</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QTableView" name="tableView"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QPushButton" name="btnStep">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>41</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>Century Gothic</family>
                <pointsize>9</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Шаг</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="ruleBox">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>Century Gothic</family>
                <pointsize>9</pointsize>
               </font>
              </property>
              <property name="title">
               <string>Правила:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_3">
              <property name="font">
               <font>
                <family>Century Gothic</family>
                <pointsize>9</pointsize>
               </font>
              </property>
              <property name="text">
               <string>   Ход разбора:</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QListView" name="listView"/>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QPushButton" name="btnNewParse">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>41</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Century Gothic</family>
              <pointsize>9</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Начать новый разбор</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnRepeat">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>41</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Century Gothic</family>
              <pointsize>9</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Просмотреть сначала</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnShowAll">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>41</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Century Gothic</family>
              <pointsize>9</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Показать разбор полностью</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnBack">
       <property name="minimumSize">
        <size>
         <width>300</width>
         <height>41</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Century Gothic</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Назад</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
